README.md

```diff
@@ -2,11 +2,45 @@
 Allows for full control of the PicoBrew Pico S/C/Pro & Zymatic models.  Shout out to [@hotzenklotz](https://github.com/hotzenklotz/picobrew-server), Brian Moineau for PicoFerm API, @tmack8001 for Z series support & updates.  
 [Demo Server](http://ec2-3-136-112-93.us-east-2.compute.amazonaws.com/)
 
+## Supported Devices
+* Pico S/C/Pro: fully featured
+* Zymatic: fully featured
+* ZSeries (Beta, with additional development requirements for SSL support)
+  * Working (with limited testing)
```

Comment:
Since you are cleaning things up (thanks!) Remove the "with limited testing" and probably the "working" type language as this has been tested well now with several different folks and for me over a month.

I never got the point of documenting what "isn't working" lol.

Reply:
Got it. Removed the "working" section. Thinking of the "Z" as generally supported now. Guessing there may be a few issues remaining with the Z, but assuming that if there are, they'll be filed.